src/azure-cli-core/azure/cli/core/sdk/util.py

```diff
@@ -57,7 +57,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         pass
 
     def command(self, name, method_name, transform=None, table_transformer=None, confirmation=None,
-                exception_handler=None, deprecate_info=None, no_wait_param=None):
+                exception_handler=None, no_wait_param=None):
```

Comment:
It doesn't make sense to revert these changes. It's optional parameters. Rather than just straight attempting to revert the commit, take some time to remove the deprecation string from the sf commands.

Reply:
no, this is introduced by this change, prefer to revert at once.